packages/components/pie-modal/test/component/pie-modal.spec.ts

```diff
@@ -373,3 +374,49 @@ test.describe('`isDismissible` prop', () => {
         });
     });
 });
+
+test.describe('isOpen prop', () => {
+    test('should not render when isOpen = false', async ({ mount, page }) => {
+        await mount(PieModal, {
+            props: {
+                isOpen: false,
+            },
+        });
+
+        await expect(page.locator(modalSelector)).not.toBeVisible();
+    });
+
+    test('should not render when isOpen = true', async ({ mount, page }) => {
+        await mount(PieModal, {
+            props: {
+                isOpen: true,
+            },
+        });
+
+        await expect(page.locator(modalSelector)).toBeVisible();
+    });
+});
+
+test('Should not be able to scroll when modal is open', async ({ page, mount }) => {
+    const modalComponent = renderTestPieModal();
+
+    await mount(
+        WebComponentTestWrapper,
+        {
+            props: {
+                pageMode: true,
+            },
+            slots: {
+                component: modalComponent,
+                pageMarkup: createScrollablePageHTML(),
+            },
+        },
+    );
+
+    // Scroll 800 pixels down the page
+    await page.mouse.wheel(0, 800);
+
+    await page.waitForTimeout(3000); // The mouse.wheel function causes scrolling, but doesn't wait for the scroll to finish before returning.
+
+    await expect(page.locator(modalSelector)).toBeInViewport();
```

Comment:
This confuses me a bit. Even if you could scroll, the modal would still be in the viewport wouldn't it?

Reply:
Exactly. But we want to test the act of scrolling to signify that we're making sure that even if it did scroll, it would still display the modal in the viewport. 
So if something changed and broke the modal being locked in place, this test would cover that.

Reply:
So my concern is that I don't think this test verifies that you cannot scroll the page. With scrolling enabled/disabled the modal is always in viewport. So how is this testing that we cannot scroll?

Reply:
Ohhh, I think I misunderstood this test. I think what we can do here is that we can test that X list item is not in the viewport when we scroll. I can change that.

Reply:
I've pushed a change now that rectifies this test. Essentially, I added a last item in the list for it to assert against if it is/isn't in the viewport after scrolling. To expand the test to be more thorough, I also added a soft assert on the top item to check it's still in view and for the last item to not be in view.